mlfinlab/backtest_statistics/statistics.py

```diff
@@ -64,7 +64,7 @@ def average_holding_period(target_positions: pd.Series) -> float:
     # Time elapsed from the starting time for each position
     time_difference = (target_positions.index - target_positions.index[0]) / \
```

Comment:
You code is too squashed. Please extend your line length then adjust the code. I think we use 160, if thats too long then I think its 140. Right now your code is so tight that its making lines that should be on one line, overlap to the second line.

Reply:
I was trying to fit into 80 so that it would be displayed in GitHub in one line. Will expand to 140